fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GitHubProperties.java

```diff
@@ -29,5 +29,7 @@
   @Min(1L)
   Integer paginationValue = 100;
   @NotNull
-  Integer membershipCacheTTLSeconds = 60 * 60; // 1 hour
+  Integer membershipCacheTTLSeconds = 60 * 10; // 10 min time to refresh
```

Comment:
Is there a reason to bring this TTL down? Wouldn't a lower TTL cause more requests?

Reply:
Previously loadRoles for each users would make 1 call to verify org membership, 1 call to get all teams for the org and 1 call per team to check team membership.   So for 100 users and 100 orgs the initial cache load would result in 10020 calls to github which would fire around once an hour.   

With the introduction of the teamsCache and the membersCache along with the change of the CacheKey to just be the teamId the same number of users and teams result in 102 calls to github.   As a result I felt it would be ok to reduce the CacheTTL used for the refresh eligibility.